test/test_methods_test.rb

```diff
@@ -11,6 +11,18 @@ def test_asserts_controller_submits_correct_form
     end
   end
 
+  def test_asserts_controller_submits_at_all
+    runner = FooControllerTest.new :foo
+
+    error = assert_raises MiniTest::Assertion do
+      runner.assert_submits FooForm do
+      end
+    end
+
+    assert_equal "Expected TestMethodsTest::FooController to submit a TestMethodsTest::FooForm",
+                 error.message
```

Comment:
I can't find much info on this type of situation. That makes sense to me.
https://stackoverflow.com/questions/12083011/how-to-break-up-long-lines-of-ruby